apps/client/src/features/modals/EventSettingsModal.jsx

```diff
@@ -99,19 +103,19 @@ export default function SettingsModal() {
           </div>
           <div className={style.hSeparator}>Additional Screen Info</div>
           <div className={style.spacedEntry}>
-            <FormLabel htmlFor='url'>
-              Event URL
+            <FormLabel htmlFor='publicUrl'>
+              Public URL
               <span className={style.labelNote}>
                 <br />
-                Shown as a QR code in some views
+                QR code to be shown on public screens
               </span>
             </FormLabel>
             <Input
               {...inputProps}
-              name='url'
+              name='publicUrl'
               placeholder='www.onsite.no'
```

Comment:
Wow. This is some old code...

Can you please update this to redirect to the ontime website?

Www.getontime.no

Make sure to update all references